app/views/content_items/service_manual_service_standard.html.erb

```diff
@@ -47,7 +47,7 @@
     </div>
 
     <div class="govuk-grid-column-one-third">
-      <aside class="related govuk-!-padding-top-4">
```

Comment:
Just curious why the classes have been removed here? Were they not needed?

Reply:
My mistake, I initially put the GA4 tracking attributes on the `<aside>` but then I moved them into the `email_signup` partial that is called on the next line. It looks like I was a bit careless and deleted the class when I deleted the attributes. Thank you for spotting that!